src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
+        return new Ball(new ColorSupplier().getRandomColor(), value);
```

Comment:
same here but also with the `colorSupplier`, we should reuse both vars here, and not create a new ones each time when `getRandomBall()` method is called

Reply:
fixed and add some changes for generate nextInt(100) + 1;  because before, you had the opportunity to obtain the ball with the number 0